repo_fetcher.py

```
@@ -88,25 +137,84 @@ def get_repo_contents(self, repo: Repository, path: str = "") -> List[Dict]:
+                                    file_content = file.decode()
+                                
+                                contents.append({
+                                    'name': item['name'],
+                                    'path': item['path'],
+                                    'type': 'file',
+                                    'size': item['size'],
+                                    'content': file_content
+                                })
+                elif platform == 'bitbucket':
+                    # Use Bitbucket API to get repository contents
+                    repo_data = repo.get('repo_data')
```

Ruff (0.14.13)

Local variable `repo_data` is assigned to but never used

Remove assignment to unused variable `repo_data`

(F841)